requirements.txt

```diff
@@ -8,9 +8,27 @@ django-debug-toolbar==2.0
 codecov==2.0.15
 coverage==4.5.3
 wagtailfontawesome>=1.1.3,<1.2
+wagtail-linkchecker==0.4.0
 django-widget-tweaks==1.4.5
+docopt==0.6.2
+draftjs-exporter==2.1.6
+gunicorn==19.9.0
+html5lib==1.0.1
+idna==2.8
+importlib-metadata==0.23
+isort==4.3.21
+kombu==4.6.4
+lazy-object-proxy==1.4.2
+mccabe==0.6.1
+more-itertools==7.2.0
+num2words==0.5.10
+elasticsearch>=6.0.0,<6.3.1
+Pillow==5.4.1
+pkg-resources==0.0.0
+psycopg2==2.8.2
+pylint==2.3.1
 python-decouple==3.1
 num2words==0.5.10
 beautifulsoup4==4.6.0
 bleach==3.1.0
-elasticsearch>=6.0.0,<6.3.1
\ No newline at end of file
+celery>=4.2.0rc4
```

Comment:
Instead of a release candidate, why not version 4.3.0? https://pypi.org/project/celery/

Reply:
Just tested with 4.3.0 and it works as well. It has been added and commited to the requirements.txt